packages/fiori/src/SideNavigationItemBase.ts

```diff
@@ -0,0 +1,241 @@
+ *
+ * @event sap.ui.webc.fiori.SideNavigationItemBase#click
+ * @public
+ */
+@event("click")
+class SideNavigationItemBase extends UI5Element implements ITabbable {
+	/**
+	 * Defines the text of the item.
+	 *
+	 * @public
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.fiori.SideNavigationItemBase.prototype.text
+	 */
+	@property()
+	text!: string;
+
+	/**
+	 * Defines the icon of the item.
+	 * <br><br>
+	 *
+	 * The SAP-icons font provides numerous options.
+	 * <br>
+	 * See all the available icons in the <ui5-link target="_blank" href="https://sdk.openui5.org/test-resources/sap/m/demokit/iconExplorer/webapp/index.html">Icon Explorer</ui5-link>.
+	 * @public
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.fiori.SideNavigationItemBase.prototype.icon
+	 */
+	@property()
+	icon!: string;
+
+	/**
+	 * Defines whether the item is selected
+	 *
+	 * @public
+	 * @type {boolean}
+	 * @defaultvalue false
+	 * @name sap.ui.webc.fiori.SideNavigationItemBase.prototype.selected
+	 */
+	@property({ type: Boolean })
+	selected!: boolean;
+
+	/**
+	 * Defines the link target URI. Supports standard hyperlink behavior.
+	 * If a JavaScript action should be triggered,
+	 * this should not be set, but instead an event handler
+	 * for the <code>click</code> event should be registered.
+	 *
+	 * @public
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.fiori.SideNavigationItemBase.prototype.href
+	 * @since 1.19.0
+	 */
+	@property()
+	href!: string;
+
+	/**
+	 * Defines the component target.
+	 * <br><br>
+	 * <b>Notes:</b>
+	 *
+	 * <ul>
+	 * <li><code>_self</code></li>
+	 * <li><code>_top</code></li>
+	 * <li><code>_blank</code></li>
+	 * <li><code>_parent</code></li>
+	 * <li><code>_search</code></li>
+	 * </ul>
+	 *
+	 * <b>This property must only be used when the <code>href</code> property is set.</b>
+	 *
+	 * @public
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.fiori.SideNavigationItemBase.prototype.target
+	 * @since 1.19.0
+	 */
+	@property()
+	target!: string;
+
+	/**
+	 * Defines whether the component is disabled.
+	 * A disabled component can't be pressed or
+	 * focused, and it is not in the tab chain.
+	 *
+	 * @type {boolean}
+	 * @name sap.ui.webc.main.SideNavigationItemBase.prototype.disabled
+	 * @defaultvalue false
+	 * @public
+	 * @since 1.19.0
+	 */
+	@property({ type: Boolean })
+	disabled!: boolean;
+
+	/**
+	 * Defines the tooltip of the component.
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @private
+	 * @name sap.ui.webc.fiori.SideNavigationItemBase.prototype.title
+	 * @since 1.0.0-rc.16
+	 */
+	@property()
+	title!: string;
+
+	@property({ defaultValue: "-1", noAttribute: true })
+	_tabIndex!: string;
+
+	get _tooltip() {
+		return this.title || this.text;
+	}
+
+	get _href() {
+		return (!this.disabled && this.href) ? this.href : undefined;
+	}
+
+	get _target() {
+		return (!this.target && this.target) ? this.target : undefined;
```

Comment:
This check seems wrong.
Given the markup:
```html
<ui5-side-navigation-sub-item href="https://sap.com" target="_blank" text="SAP"></ui5-side-navigation-sub-item>
```
Clicking it should open a new tab, but it doesn't do it.
Did you mean to write `(!this.disabled && this.target)`?